MQuince/MQuince.WebAPI/Controllers/AppointmentController.cs

```diff
@@ -112,5 +112,30 @@ public IActionResult Update(Appointment appointment)
             _appointmentService.Update(new AppointmentDTO() { StartDateTime = appointment.StartDateTime, EndDateTime = appointment.EndDateTime, Type = appointment.Type, DoctorId = appointment.DoctorId, PatientId = appointment.PatientId }, appointment.Id);
             return Ok();
         }
+
+        [HttpPut("canceledAppointment/{IdAppointment}")]
```

Comment:
Replace path to: cancelAppointment 

Method parameters are written in lower case.
example: appointmentId